.github/workflows/docs.yaml

```
@@ -0,0 +1,31 @@
+name: docs
+on:
+  push:
+    branches:
+    - conformance-1.2.1
+    paths:
+    # Only rebuild website when docs have changed
+    - 'README.md'
+    - 'docs/**'
+    - 'mkdocs.yml'
+    - .github/**
```

probably better to build only on release
```suggestion
  release:
    types: [created]
```

---

agreed